test/infra/anthos-managed/namespaces/secretmanager-csi-build/integration-test-resources.yaml

```diff
@@ -0,0 +1,55 @@
+  # Copyright 2020 Google LLC
+  #
+  # Licensed under the Apache License, Version 2.0 (the "License");
+  # you may not use this file except in compliance with the License.
+  # You may obtain a copy of the License at
+  #
+  #     http://www.apache.org/licenses/LICENSE-2.0
+  #
+  # Unless required by applicable law or agreed to in writing, software
+  # distributed under the License is distributed on an "AS IS" BASIS,
+  # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  # See the License for the specific language governing permissions and
+  # limitations under the License.
+apiVersion: iam.cnrm.cloud.google.com/v1beta1
+kind: IAMServiceAccount
+metadata:
+  name: k8s-csi-test
+spec:
+  displayName: Integration tests Service Account
+---
+apiVersion: secretmanager.cnrm.cloud.google.com/v1beta1
+kind: SecretManagerSecret
+metadata:
+  name: test-secret-a
+  labels:
+    replication-type: automatic
+spec:
+  replication:
+    automatic: true
+---
+  apiVersion: secretmanager.cnrm.cloud.google.com/v1beta1
+  kind: SecretManagerSecretVersion
+  metadata:
+    name: test-secret-a-version
+  spec:
+    enabled: true
+    secretData:
+      # "hunter2"
+      value: "aHVudGVyMg=="
+    secretRef:
+      name: test-secret-a
+---
+apiVersion: iam.cnrm.cloud.google.com/v1beta1
+kind: IAMPolicy
+metadata:
+  name: test-secret-a-binding
+spec:
+  resourceRef:
+    apiVersion: kms.cnrm.cloud.google.com/v1beta1
```

Comment:
should be `secretmanager.cnrm.cloud.google.com/v1beta1` right? Also why not v1?

Reply:
Ah thanks. I think its `v1beta1` because its representative of the CRD API _not_ the actual secretmanager API version.

The CRD info came from https://cloud.google.com/config-connector/docs/reference/resource-docs/secretmanager/secretmanagersecretversion